imports/plugins/included/default-theme/client/styles/cart/checkout.less

```diff
@@ -230,7 +230,6 @@
 .order-details-media {
   width: 40px;
   height: 40px;
-  border: solid 1px #cccccc;
```

Comment:
Is removing the border here a new design direction we're taking? I'd thought we wanted the border around the images?

![screenshot_2018-02-15_14_37_03_and_screenshot_2018-02-15_14_37_12_and_screenshots_and_test](https://user-images.githubusercontent.com/5229321/36259300-15d4e760-125e-11e8-99df-4fa825e9bede.jpg)